src/main/java/org/springframework/data/redis/core/RedisOperations.java

```diff
@@ -160,17 +160,17 @@ <T> T execute(RedisScript<T> script, RedisSerializer<?> argsSerializer, RedisSer
 	// -------------------------------------------------------------------------
 
 	/**
-	 * Copy given {@code sourceKey} to {@code targetKey}.
+	 * Copies the value stored at the {@code source} to the {@code destination}.
 	 *
-	 * @param sourceKey must not be {@literal null}.
-	 * @param targetKey must not be {@literal null}.
+	 * @param source must not be {@literal null}.
+	 * @param destination must not be {@literal null}.
```

Comment:
Now, neither `source` nor `destination` describe what they are. For instance, are they keys, are they values, what?

"`must not be {@literal null}`" is not informative and arguably no longer needed given the package-level `@NonNullApi` requires non-`null` values when method parameters are not explicitly annotated with `@Nullable`.

I am fine with shorter names as long as the Javadoc makes it apparent what things are and what they are used for.